content/docs/hello-world.md

```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+Ipinapakita rito ang isang heading na nagsasabing "Hello, world!" sa pahina.
```

Comment:
We should proceed making this "Kamusta, mundo!", what you think? 😉

Reply:
Ow I didn't translate "Hello, world" cause it is linked to a codepen 😅 which says 

![image](https://user-images.githubusercontent.com/23741509/95698774-44ce2680-0c75-11eb-9c8a-6e0b59130da1.png)

![image](https://user-images.githubusercontent.com/23741509/95698780-48fa4400-0c75-11eb-8ea6-c62bbb44b3ec.png)

So I left it at that cause I can't edit the CodePen.

Moreover, when I checked the other React docs translation like Spanish.

![image](https://user-images.githubusercontent.com/23741509/95698855-7515c500-0c75-11eb-8add-48ac86b257ab.png)

They left it at Hello World too.